### PR TITLE
[IMP] add transifex support and update .travis.yml to latest MQT template

### PR DESCRIPTION
**Pull request by sbidoul**
align travis.yml to latest MQT template

see also https://github.com/OCA/maintainer-quality-tools/issues/194

### CODE REVIEW

**Comment by lmignon** [comment]
In the same time you should also add a 'translation status' section in README.rst https://raw.githubusercontent.com/OCA/maintainer-quality-tools/master/sample_files/README.md

**Comment by sbidoul** [comment]
@lmignon done

**Comment by lmignon** [comment]
:+1:  Thank you!

File: README.md [hunk 1]
```diff
@@ -1,7 +1,15 @@
+[![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/123/8.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-l10n-luxemburg-123)
 [![Build Status](https://travis-ci.org/OCA/l10n-luxemburg.svg?branch=8.0)](https://travis-ci.org/OCA/l10n-luxemburg)
-[![Coverage Status](https://coveralls.io/repos/OCA/l10n-luxemburg/badge.png?branch=8.0)](https://coveralls.io/r/OCA/l10n-luxemburg?branch=8.0)
+[![Coverage Status](https://coveralls.io/repos/OCA/l10n-luxemburg/badge.svg?branch=8.0)](https://coveralls.io/r/OCA/l10n-luxemburg?branch=8.0)
+[![Code Climate](https://codeclimate.com/github/OCA/{REPO_NAME}/badges/gpa.svg)](https://codeclimate.com/github/OCA/l10n-luxemburg)
```
Comment (pedrobaeza) [inline README.md]: Change `{REPO_NAME}` to `l10-luxemburg`

File: README.md [hunk 2]
```diff
@@ -1,7 +1,15 @@
+[![Runbot Status](https://runbot.odoo-community.org/runbot/badge/flat/123/8.0.svg)](https://runbot.odoo-community.org/runbot/repo/github-com-oca-l10n-luxemburg-123)
```
Comment (pedrobaeza) [inline README.md:1]: This flag is not working

Comment (laetitia-gangloff) [inline README.md:1]: Hmm... I see this flag and can click on it. What do you mean ?

**Comment by laetitia-gangloff** [comment]
Hello,

I rebased the branch and corrected the readme.md.

**Comment by sbidoul** [comment]
I think I can self merge this one, as this is repository maintenance.

**Pull request merged**